project/src/main/ui/chat/chat-tree.gd

```diff
@@ -28,14 +28,9 @@ const CHAT_DATA_VERSION := "1922"
 const DEFAULT_ENVIRONMENT := "res://src/main/world/environment/EmptyEnvironment.tscn"
 
 const ENVIRONMENT_SCENE_PATHS_BY_ID := {
-	"indoors": "res://src/main/world/environment/OverworldIndoorsEnvironment.tscn",
-	"outdoors": "res://src/main/world/environment/OverworldEnvironment.tscn",
-	"outdoors_walk": "res://src/main/world/environment/OverworldWalkEnvironment.tscn",
-	
-	"marsh/walk": "res://src/main/world/environment/OverworldWalkEnvironment.tscn",
-	"marsh/inside_turbo_fat": "res://src/main/world/environment/OverworldIndoorsEnvironment.tscn",
-	"marsh/outside_turbo_fat": "res://src/main/world/environment/marsh/MarshEnvironment.tscn",
-	"marsh/buttercup_cafe": "res://src/main/world/environment/marsh/MarshEnvironment.tscn",
+	"marsh": "res://src/main/world/environment/MarshEnvironment.tscn",
+	"marsh/walk": "res://src/main/world/environment/MarshWalkEnvironment.tscn",
```

Comment:
Should be `environment/marsh/MarshWalkEnvironment`. Regression test cutscenes in cutscene demo.

Reply:
Fixed scene paths. Regression tested an outdoors cene and the epilogue scene.